cli/remote_build.go

```diff
@@ -348,13 +342,17 @@ func (p *RemoteCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...interface{}
 	return subcommands.ExitSuccess
 }
 
-func commitTempChanges(w *git.Worktree) (latest plumbing.Hash, err error) {
+func commitTempChanges(w *git.Worktree, c *object.Commit) (latest plumbing.Hash, err error) {
+	if w == nil || c == nil {
+		err = fmt.Errorf("Needs a worktree and a commit object")
+		return
+	}
 	latest, err = w.Commit(
 		"YourBase remote build",
 		&git.CommitOptions{
 			Author: &object.Signature{
-				Name:  "YourBase",
-				Email: "robot@yourbase.io",
+				Name:  c.Author.Name,
```

Comment:
You asked me that a loooooong time ago :laughing:

Reply:
:+1: